bitnami/drupal/README.md

```diff
@@ -52,140 +52,140 @@ The following table lists the configurable parameters of the Drupal chart and th
+| `mariadb.enabled`                           | Whether to use the MariaDB chart                                                                                      | `true`                                                       |
+| `mariadb.architecture`                      | MariaDB architecture (`standalone` or `replication`)                                                                  | `standalone`                                                 |
+| `mariadb.auth.rootPassword`                 | Password for the MariaDB `root` user                                                                                  | _random 10 character alphanumeric string_                    |
+| `mariadb.auth.database`                     | Database name to create                                                                                               | `bitnami_drupal`                                             |
+| `mariadb.auth.username`                     | Database user to create                                                                                               | `bn_drupal`                                                  |
+| `mariadb.auth.password`                     | Password for the database                                                                                             | _random 10 character long alphanumeric string_               |
+| `mariadb.primary.persistence.enabled`       | Enable database persistence using PVC                                                                                 | `true`                                                       |
+| `mariadb.primary.persistence.existingClaim` | Name of an existing `PersistentVolumeClaim` for MariaDB primary replicas                                              | `nil`                                                        |
+| `mariadb.primary.persistence.accessModes`   | Database Persistent Volume Access Modes                                                                               | `[ReadWriteOnce]`                                            |
+| `mariadb.primary.persistence.size`          | Database Persistent Volume Size                                                                                       | `8Gi`                                                        |
+| `mariadb.primary.persistence.storageClass`  | PVC Storage Class                                                                                                     | `nil` (uses alpha storage class annotation)                  |
+| `mariadb.primary.persistence.storageClass`  | MariaDB primary persistent volume storage Class                                                                       | `nil`                                                        |
```

Comment:
This should be `mariadb.master.persistence.hostPath`. It looks like a typo as there is two entries for `storageClass`.

Reply:
yes it should be `mariadb.primary.persistence.hostPath`, thanks!